src/services/projection.service.ts

```diff
@@ -0,0 +1,37 @@
+import { get as getProjection } from 'ol/proj'
+import { register } from 'ol/proj/proj4'
+import proj4 from 'proj4'
+
+class ProjectionService {
+  initProjections() {
```

Comment:
Do I see right, that these projections are needed in the `StatePersistorMapService` when getting the view center? Would it make sense to export it as a simple utility function instead of a service?

Reply:
yes you are right, let's change that service into a utility fn.